### PR TITLE
AvalancheGo@v1.7.17

### PR DESCRIPTION
**Pull request by rajranjan0608**
AvalancheGo -> 1.7.17
Coreth -> 0.8.15

### CODE REVIEW

**Review by patrick-ogrady — APPROVED** [review]

File: service/rosetta.go [hunk 1]
```diff
@@ -8,9 +8,9 @@ import (
 
 var NodeVersion = fmt.Sprintf(
 	"%d.%d.%d",
-	version.Current.Major(),
-	version.Current.Minor(),
-	version.Current.Patch(),
+	version.Current.Major,
+	version.Current.Minor,
+	version.Current.Patch,
```
Comment (StephenButtolph) [inline service/rosetta.go:13]: Are we supposed to bump the `MiddlewareVersion` too?

Comment (patrick-ogrady) [reply service/rosetta.go:13]: I usually do that once this is merged.

Comment (patrick-ogrady) [reply service/rosetta.go:13]: but probably better for @rajranjan0608 to do it here.

Comment (rajranjan0608) [reply service/rosetta.go:13]: Sure thing.

Comment (patrick-ogrady) [reply service/rosetta.go:13]: https://github.com/ava-labs/avalanche-rosetta/blob/05c01b41a3dcad4c217212e49fbeb4f25311bb91/service/rosetta.go#L17

**Review by StephenButtolph — APPROVED** [review]

**Review by dhrubabasu — APPROVED** [review]

**Review by patrick-ogrady — APPROVED** [review]

**Pull request merged**